source/causal-consistency/causal-consistency.md

```diff
@@ -0,0 +1,422 @@
+# Causal Consistency Specification
+
+- Status: Accepted
+- Minimum Server Version: 3.6
+
+______________________________________________________________________
+
+## Abstract
+
+Version 3.6 of the server introduces support for causal consistency. This spec builds upon the Sessions Specification to
+define how an application requests causal consistency and how a driver interacts with the server to implement causal
+consistency.
+
+## Definitions
+
+### META
+
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
+“OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+### Terms
+
+**Causal consistency**\
```

Comment:
Is this how you've been manually converting definition lists? Makes sense here, as it's probably the closest formatting we have to the RST.

Reply:
Yes